### ISSUE TITLE
Add the ability to use one of several preset banks

### ISSUE THREAD

**Issue by GoogleCodeExporter** [400plus/400plus#298]
```
Currently, to use more than the allowable nine presets, we must create the 
presets on additional cards.

A more convenient mechanism would be to allow multiple preset banks on a single 
card. This might work as follows:

In the Settings-->Config. Presets menu, we could have an option named "Preset 
Bank." The user can select from say, one of three banks (The presets for scene 
modes (issue #135) could be in a specially named bank).

After specifying a preset bank, the preset menu title will show "Presets - 
Bankx," where x represents the bank number. The user configures the presets in 
a bank as he does currently.

Notes:
1.Only one bank will be active at a time. To switch banks, simply select a new 
bank in the Settings-->Config. Presets menu.

2.It should be possible to copy a preset from one bank to another.

3.Presumably, each bank would represent separate folders on the card.

As a side note, is there a reason why there can't be ten presets instead of 
nine? I ask this because a screen displays five presets.
```

Original issue reported on code.google.com by `colinban...@gmail.com` on 22 Nov 2012 at 2:06
- Blocked on: #135
- Merged into: #304
